Util/Factory/Query/DoctrineBuilder.php

```diff
@@ -153,13 +153,24 @@ protected function _addSearch(\Doctrine\ORM\QueryBuilder $queryBuilder, array $f
                         $search_field = $filter_fields[$i]->getSearchField();
                         $parts = explode(',', $search_param);
 
-                        $ors = [];
-                        foreach ($parts as $k=>$part)
+                        if (\count($parts) === 1)
                         {
-                            $ors[] = "$search_field = :ssearch{$i}_part{$k}";
-                            $queryBuilder->setParameter("ssearch{$i}_part{$k}", trim($part));
+                            $first_part = reset($parts);
+
+                            $queryBuilder->andWhere("$search_field = :ssearch{$i}");
+                            $queryBuilder->setParameter("ssearch{$i}", trim($first_part));
```

Comment:
even kijken of die trim logisch is. wellicht wel met hoe het opgestuurd wordt.

Reply:
Yea zou de huidige werking niet aanpassen :)